README.md

```diff
@@ -53,10 +53,10 @@ Done predicting. Predict table : iris.predict
 - [Installation](doc/installation.md)
 - [Running a Demo](doc/demo.md)
 - [Extended SQL Syntax](doc/syntax.md)
+- [Build from source code](doc/build.md)
```

Comment:
Users don't need to build from source, only contributors need to.

Reply:
Users and developers seem be inter-changable,  I agree we can keep it under contribution.